docs/advanced/using-testnet.md

```diff
@@ -39,3 +39,16 @@ To issue commands to both `dcrwallet` and `dcrd`, you must also add the `--testn
 ## Acquiring Testnet Coins
 
 You can acquire coins through the [Decred Testnet Faucet](https://faucet.decred.org). Please return any coins to the address listed at the bottom of that page when you're done playing with the testnet.
+
+---
+
+## Voting Service Providers (VSP)
+
+To experiment with staking and voting with a Voting Service Provider (VSP), you'll need to sign up for a testnet VSP. 
+
+### Testnet VSPs
```

Comment:
I don't think we need two headings for this, what do you think about updating the first one to say "Testnet Voting Service Providers (VSP)" and removing the second one?

Reply:
Was on the fence about that myself. Have implemented this suggestion. Looks better.